usr/src/cmd/mdb/common/mdb/mdb_fmt.c

```diff
@@ -798,13 +798,22 @@ mdb_fmt_print(mdb_tgt_t *t, mdb_tgt_as_t as,
 				rvalue = u.i4;
 				break;
 			case 8:
+				/*
+				 * XXXKDI: cannot even refer to floating point
```

Comment:
Do we need fp context save and restore? Floating point is always present in some form on armv8+.

Reply:
The arm64 gcc and ARM architecture are very literal about not using floating point (like in the kernel and kmdb).  If we say "Don't use floating point", you can't even mention a floating point type.  It is the equivalent of the FPU being turned off.